lib/src/chart/base/axis_chart/axis_chart_data.dart

```diff
@@ -326,6 +326,7 @@ class SideTitles with EquatableMixin {
   final double? interval;
   final double rotateAngle;
   final CheckToShowTitle checkToShowTitle;
+  final TextAlign textAlign;
```

Comment:
Or you can declare it above the `checkToShowTitle`. (everywhere you put it above `checkToShowTitle`)

Reply:
Yeah, that's what I meant to do but missed that one, Good Catch :)